styles/globals.css

```diff
@@ -7,3 +7,8 @@ body {
 * {
   box-sizing: border-box;
 }
+
+/* TODO (summer-cook) remove this style after adding bootstrap to component library */
+.text-end {
+  text-align: right;
+}
```

Comment:
can you move this to the component library globals.css instead?

Reply:
Sure, heres the PR for it in component lib: https://github.com/scientist-softserv/webstore-component-library/pull/59